edit/service.vue

```diff
@@ -150,9 +175,46 @@ export default {
     const initialType = this.serviceType;
 
     this.$set(this, 'serviceType', initialType);
+
+    this.podPoller.start();
+  },
+
+  beforeDestroy() {
+    this.podPoller.stop();
   },
 
   methods: {
+    udpateMatchingPods: throttle(function() {
+      const { allPods, value: { spec: { selector = { } } } } = this;
+
+      if (isEmpty(selector)) {
+        this.matchingPods = {
+          matched: 0,
+          total:   allPods.length,
+          none:    true,
+          sample:  null,
+        };
+      } else {
+        const match = matching(allPods, selector);
+
+        this.matchingPods = {
+          matched: match.length,
+          total:   allPods.length,
+          none:    match.length === 0,
+          sample:  match[0] ? match[0].nameDisplay : null,
+        };
+      }
+    }, 250, { leading: true }),
+
+    async loadPods() {
+      try {
+        const inStore = this.$store.getters['currentProduct'].inStore;
+
+        this.allPods = await this.$store.dispatch(`${ inStore }/findAll`, { type: POD });
+        this.matchingPods.total = this.allPods.length;
+      } catch (e) { }
+    },
```

Comment:
I don't see why this needs to be periodically polled?  The find doesn't have force so it's getting back the same array every time, and allPods.length should be "live".